app/src/main/java/com/island/island/Utils/ImageUtils.java

```diff
@@ -4,34 +4,71 @@
 import android.content.res.AssetManager;
 import android.graphics.Bitmap;
 import android.graphics.BitmapFactory;
+import android.net.Uri;
+import android.provider.MediaStore;
+import android.util.Log;
+import android.widget.ImageView;
 
+import com.island.island.R;
+
+import java.io.ByteArrayOutputStream;
 import java.io.File;
 import java.io.FileInputStream;
+import java.io.FileNotFoundException;
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 
-/**
- * Created by poo on 2/10/2016.
- */
 public class ImageUtils
 {
+    private static String TAG = ImageUtils.class.getSimpleName();
     private static String IMAGE_DIR = "images";
+    private static String JPG_EXT = ".jpg";
+
+    private static String DEFAULT_HEADER_ASSET = "default_header.jpg";
+    private static String DEFAULT_PROFILE_ASSET = "default_profile.jpg";
+
+    private static String SLASH = "/";
+
+    private static int COMPRESSION = 65;
 
     public static void saveBitmapToInternalStorage(Context context, Bitmap bitmap, String filePath)
     {
         File directory = context.getDir(IMAGE_DIR, Context.MODE_PRIVATE);
         File bitmapPath = new File (directory, filePath);
-        FileOutputStream outputStream = null;
 
         try
         {
-            outputStream = new FileOutputStream(bitmapPath);
-            bitmap.compress(Bitmap.CompressFormat.PNG, 100, outputStream);
+            FileOutputStream outputStream = new FileOutputStream(bitmapPath);
+            bitmap.compress(Bitmap.CompressFormat.JPEG, COMPRESSION, outputStream);
+            try
+            {
+                outputStream.close();
+            }
+            catch (IOException e)
+            {
+                e.printStackTrace();
```

Comment:
Yeah a lot of logging code needs to be added. Think I might do this in the clean up PR